README.md

```diff
@@ -24,10 +19,6 @@ Follow the [Set up environment](https://learn.microsoft.com/training/modules/int
     ├── libs                         # Submoduled ThreadX and NetX Duo source code
     └── tools                        # Required scripts for using NetX Duo within the container
 
-### Use Visual Studio
```

Comment:
is there a reason this was deleted?

Reply:
This section is part of the content that is missing from the download, and would need the zip package to be rebuilt by the team with an .sln file.  It's one part of the items that were tied to the RTOS training content and didn't come through in the repo move.